app/controllers/api/v1/api_base_controller.rb

```diff
@@ -26,9 +26,11 @@ class ApiBaseController < ActionController::API
         ErrorTracker.report_exception(exception, current_api_user, params)
         logger.error ([exception.message] + exception.backtrace).join($/)
         if Rails.env.production?
-          render json: {error: "An unexpected error has occurred"}, status: 500
+          render json: {error: "An unexpected error has occurred"}, status: 500 and return
```

Comment:
I think we actually proved that these don't need to be here, but they certainly don't hurt anything.

Reply:
concur